services/fastmail/config.json

```diff
@@ -19,16 +27,114 @@
       "prio": 20
     },
     {
+      "name": "*",
+      "type": "MX",
+      "content": "in1-smtp.messagingengine.com",
+      "ttl": 3600,
+      "prio": 10
+    },
+    {
+      "name": "*",
+      "type": "MX",
+      "content": "in2-smtp.messagingengine.com",
+      "ttl": 3600,
+      "prio": 20
+    },
+    {
+      "name": "mail",
+      "type": "MX",
+      "content": "in1-smtp.messagingengine.com",
+      "ttl": 3600,
+      "prio": 10
+    },
+    {
+      "name": "mail",
+      "type": "MX",
+      "content": "in2-smtp.messagingengine.com",
+      "ttl": 3600,
+      "prio": 20
+    },    {
```

Comment:
The opening `{` should be moved to the next line to follow the style of the rest of this file.